### PR TITLE
[CTCP-4667] remove additional reference page for houseconsignment

### PR DESCRIPTION
**Pull request by prashanthmamidi**

### CODE REVIEW

File: app/controllers/houseConsignment/index/items/additionalReference/RemoveAdditionalReferenceYesNoController.scala [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    extends FrontendBaseController
+    with I18nSupport {
+
+  private val form = formProvider("houseConsignment.index.items.additionalReference.removeAdditionalReferenceYesNo")
+
+  private def addAnother(arrivalId: ArrivalId, mode: Mode): Call =
+    Call("GET", "#")
+  // TODO: update with AddAnotherAdditionalReferenceController
+
+  def insetText(userAnswers: UserAnswers, houseConsignmentIndex: Index, itemIndex: Index, additionalReferenceIndex: Index): Option[String] = {
```
Comment (joecefnhedog) [inline app/controllers/houseConsignment/index/items/additionalReference/RemoveAdditionalReferenceYesNoController.scala:52]: Does this need to be an Option if we always return a `Some(additionalReferenceType + insetTestAddRef).

Comment (prashanthmamidi) [reply app/controllers/houseConsignment/index/items/additionalReference/RemoveAdditionalReferenceYesNoController.scala:52]: @joecefnhedog  yeah, it return none when there's no additionalReferenceType- updated the code

**Comment by prashanthmamidi** [comment]
image

**Comment by prashanthmamidi** [comment]
image

**Pull request merged**